### DISCUSSION TITLE
Announcement:  New Theme Available.  Simple Classic version 1.1.0.0

### DISCUSSION THREAD

**Discussion by bjazmoore** [GetPublii/Publii#1741] [Useful Resources]
**Well - it is not a new theme really** - it is an update to a tried and true theme.  While I really love the new Simple theme that comes with Publii 0.46.x (Great work @bobmitro @dziudek), I also miss the old theme which fit a couple of my use cases really well, so...

**I opened up my editor** and went to work updating the old Simple theme.  We could not call it Simple anymore, because there is a new, better and far prettier Simple gracing Publii these days, so...

This new (old) theme is called **Simple Classic**.  Kind of like COKE Classic (anyone remember that?)

**You will have to pick up the new theme** from my [repository.](https://github.com/bjazmoore/Simple-Classic) where you can check out some screen shots of the theme in action.  I have added a couple new twists to the theme that were not part of the original Simple.

**Of course the new Simple Classic supports Pages** as well as displaying child page structures on a page.

**Want to just download it?**  Get it here: [Simple Classic 1.1.0.0](https://github.com/bjazmoore/Simple-Classic/releases/download/1.1.0.0/simple.classic.1.1.0.0.zip)

Let me know what you think.

**Comment by gpsblues** [comment]
Hi,

I also really appreciate the old Simple especially for the 100% hero that allows me to easily insert a full video in the viewport.
I had already started to convert the old theme, which I called "Simple Due", but this way I find the work already done :)

Thanks!

**Reply by bjazmoore** [reply]
Excellent.  So glad to be able to help.

**Comment by rlesyk** [comment]
Great job bjazomoore... I love the old simple theme but this, this new simple-classic them... well it already includes all that I need.
Thank you and keep it up. Starting to really love Publii, it's a game changer.

**Reply by bjazmoore** [reply]
Thank you.  This theme fit several of the sites I have designed very well, I just had to bring it up to modern standards.

**Comment by ProyectoFlower** [comment]
I was battling a lot with the hero of the current simple, but this is exactly what I was looking for, thank you very much.

**Comment by iffinland** [comment]
Very nice, im love it. Thanks YOU for work :)
